common/lib/xmodule/xmodule/templates/problem/checkboxes_response.yaml

```diff
@@ -1,24 +1,68 @@
 ---
 metadata:
-    display_name: Checkboxes
+    display_name: Checkbox
     markdown: |
-      A checkboxes problem presents checkbox buttons for student input. Students can select more than one option presented.
-      >>Select the answer that matches<<
+       Studio offers three types of problems that allow learners to select one or more options from a list that you provide.
+       - Checkbox problems allow learners to select multiple options. Learners can see all the options along with the problem text.
+       - Dropdown problems allow learners to select only one option. Learners only see all the options after they click the dropdown list.
+       - Multiple choice problems allow learners to select only one option. Learners can see all the options along with the problem text.
 
-      [x] correct
-      [ ] incorrect
-      [x] correct
+       To create a checkbox problem in the simple editor:
+       1. Enter the text of the main question between two pairs of angle brackets that point inward (>><<).
+       2. Enter the text of each answer option on a separate line.
+       3. Add brackets that contain a space before each answer option: [ ]
+       4. For each correct answer, add an x between the brackets: [x]
+       Note that a checkbox problem can have more than one correct answer.
+       5. To add an explanation, enter the text of the explanation, and then highlight the explanation and click the light bulb icon.
+
+       You can use the following example problem as a model.
+       _____________________________________________________________________________
+
+      >>The following languages are in the Indo-European family:<<
+      [x] Urdu
+      [ ] Finnish
+      [x] Marathi
+      [x] French
+      [ ] Hungarian
+
+      Note: Make sure you select all of the correct options—there may be more than one!
+
+      [explanation]
+      Urdu, Marathi, and French are all Indo-European languages, while Finnish and Hungarian are in the Uralic family.
+      [explanation]
 
 data: |
       <problem>
-        <p>A checkboxes problem presents checkbox buttons for student input. Students can select more than one option presented.</p>
-        <p>Select the answer that matches</p>
+      <p>To create a checkbox problem in the simple editor:</p>
+      <p>1. Enter the text of the main question between two pairs of angle brackets that point inward (>><<).</p>
+      <p>2. Add brackets that contain a space before each answer option: [ ]</p>
+      <p>3. For each correct answer, add an x between the parentheses: [x] </p>
+      <p>Note that a problem can have more than one correct answer.</p>
+      <p>4. To add an explanation, enter the text of the explanation, and then highlight the explanation and click the light bulb icon.</p>
+      <p>You can use the following example problem as a model.</p>
+      <p>_____________________________________________________________________________</p>
+
+      <fieldset>
```

Comment:
@cptvitamin, did I use ``<fieldset>`` correctly here? (I'll have this same question for the other two multiple choice problem types, multiplechoice.yaml and optionresponse.yaml.)